bin/git-changelog

```diff
@@ -146,6 +146,7 @@ _fetchCommitRange() {
   local start_tag="$2"
   local final_tag="$3"
 
+  # shellcheck disable=SC2086
   if [[ "$list_all" == true ]]; then
```

Comment:
Why add shellcheck disable here?

Reply:
This shellcheck comment applies to the `if` body, which contains use of variables that ate not quoted

Reply:
There are multiple violations and the body isn't very long, which is why I put a single one at the top

Reply:
> This shellcheck comment applies to the `if` body, which contains use of variables that ate not quoted

I see. Would you add a comment to indicate it's applied to the body?

Reply:
Added the comment :+1: